pkg/util/storage/manager.go

```diff
@@ -44,6 +47,20 @@ func (m *manager) BlobService(ctx context.Context, resourceGroup, account string
 		SharedAccessExpiryTime: &date.Time{Time: t.Add(24 * time.Hour)},
 	})
 	if err != nil {
+		if strings.Contains(err.Error(), "TooManyRequests") {
+			err = &api.CloudError{
+				StatusCode: http.StatusTooManyRequests,
+				CloudErrorBody: &api.CloudErrorBody{
+					Code:    api.CloudErrorCodeThrottlingLimitExceeded,
+					Message: "ThrottlingLimitExceeded",
+					Details: []api.CloudErrorBody{
+						{
+							Message: string("Requests are being throttled due to Azure Storage limits being exceeded. Please visit https://learn.microsoft.com/en-us/azure/openshift/troubleshoot#exceeding-azure-storage-limits for more details."),
```

Comment:
This error message does not need to be the same as what's emitted by the RP, and does not need to be "customer-facing". 

I would recommend changing this message to one that both SRE on-call, as well as a regular expression, can easily pick up on - using an identifiable and unique code, for example. We don't need the customer-facing URL either.

Reply:
Here we are trying to send the error message(modified) to the customer so that it will be more meaningful to them than a generic Internal Server Error.

Reply:
These messages never get propagated to the customer, and generally never leave the Hive cluster. We need Hive to identify these messages via regular expression to propagate status conditions on the Cluster Deployment

Reply:
Discussed this in sync meeting. We can proceed with this change as is, and ensure the RP identifies the following message:

`Requests are being throttled due to Azure Storage limits being exceeded. Please visit https://learn.microsoft.com/en-us/azure/openshift/troubleshoot#exceeding-azure-storage-limits for more details.`